charts/stackrox/templates/configure-stackrox-job.yaml

```diff
@@ -19,9 +19,12 @@ spec:
             - |
               test {{ .Values.verbose }} == true && set -x
 
-              # wait for operator
+              # wait for CRD
               while [ true ]; do oc get crd centrals.platform.stackrox.io; if [ $? -eq 0 ]; then break; fi ; sleep 5s; done
 
+              # wait for operator to spin-up
+              while test 0 == $(oc -n openshift-operators get pod -l app=rhacs-operator --no-headers 2>/dev/null | grep Running |  wc -l); do sleep 5; done
```

Comment:
is the pod always guaranteed to be in the `openshift-operators` namespace?

Reply:
For the moment it is, it's hard-coded in other places too